docs/user-documentation/getting-started-with-islandora-claw.md

```diff
@@ -0,0 +1,57 @@
+# Getting Started with Islandora CLAW
+
+## How to Add a User
+1. Log in under admin credentials.
+2. Click the **Manage** tab from the top toolbar.
+3. Select **People** tab from the resulting menu. 
```

Comment:
I completely agree. I mostly followed what the hack/doc participants had created in the Google Doc, and it had these inconsistencies. I was not suse which practices were preferred, but initially I wanted to just follow what they did until I got feedback from the community/committers. 

Is there a community documentation standard I should follow or a place I could look it up? For example, in some docs they format all menu or button labels in a particular style like bold or italics. In the Google Doc sometimes they used bold or sometimes they used quotes. Once I am given guidelines or if I find them on my own, I can reformat what I have done so far as well as continue the practice with the content that is left in the Google Doc. In the meantime, I will look over some of the docs on the Islandora wiki more carefully to see what has been done in the past. 

One thing I did change from the original hack/doc content was to enforce the use of the word "click" instead of "select" when the contents referred to clicking a button, tab or link.